urfave_cli_path.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+GOMODCACHE=$(go env | grep GOMODCACHE | sed 's/GOMODCACHE=//' | xargs)
+URFAVECLI_VER=( $(cat go.mod | grep github.com/urfave/cli | xargs) )
+
+URFAVECLI_PATH="$GOMODCACHE/${URFAVECLI_VER[0]}@${URFAVECLI_VER[1]}"
+
+echo "$URFAVECLI_PATH"
```

Comment:
I think that it is not necessary to have special file with this script, because it is used only in `Makefile`.

Or is this script useful for anything else?

If not, then I think that we should "move" the whole script (or the logic of the script) into the `Makefile`.

Reference: https://www.gnu.org/software/make/manual/html_node/One-Shell.html